docs/dg/dev/system-requirements/202407.0/system-requirements.md

```diff
@@ -0,0 +1,55 @@
+| OS                                        | Native: Linux                                                                                                                                                                                                                                                                                                                                                                                                  |
+| Web server                                | NginX—preferred. But any web server supporting PHP, like lighttpd, Apache, or Cherokee, works.                                                                                                                                                                                                                                                                                                             |
+| Databases                                 | MariaDB >= 10.4—preferred, PostgreSQL >=9.6, or MySQL >=5.7.                                                                                                                                                                                                                                                                                                   |
+| PHP                                       | PHP >=8.1 with the following extensions: `curl`, `json`, `mysql`, `pdo-sqlite`, `sqlite3`, `gd`, `intl`, `mysqli`, `pgsql`, `ssh2`, `gmp`, `mcrypt`, `pdo-mysql`, `readline`, `twig`, `imagick`, `memcache`, `pdo-pgsql`, `redis`, `xml`, `bz2`, and `mbstring`. For details about supported PHP versions, see [Supported Versions of PHP](/docs/dg/dev/supported-versions-of-php.html) |
+| SSL                                       | For production systems, a valid security certificate is required for HTTPS.                                                                                                                                                                                                                                                                                                                                    |
+| Redis                                     | Version >=5.0, >=6.2                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
Why are there two versions?

Reply:
Because 2 versions are supported. Please check https://github.com/spryker/docker-sdk/tree/master/generator/src/templates/service/redis

Reply:
In my understanding `>=5.0` includes `6.2`. Or does `>=5.0` mean versions `5.0-9`?

Reply:
Okay, I've adjusted these versions to be clearer as we support exactly those two. I would not use ">=5.0" to avoid any confusion and ensure users understand exactly which versions are compatible with their software